test/unit/specs/store/delegation.spec.js

```diff
@@ -209,7 +217,15 @@ describe(`Module: Delegations`, () => {
     const dispatch = jest.fn()
 
     await actions.submitDelegation(
-      { rootState: mockRootState, state, dispatch, commit: jest.fn() },
+      {
+        rootState: mockRootState,
+        getters: {
+          liquidAtoms: 1000
+        },
+        state,
+        dispatch,
+        commit: jest.fn()
```

Comment:
I think if we are not inspecting who's calling it, we can simply use a dummier empty function